static/src/stylesheets/module/facia-garnett/_pillars.scss

```diff
@@ -412,14 +412,14 @@ $pillars: (
         }
     }
 
-    background-color: $special-report-alt-faded !important;
+    background-color: $special-report-alt-faded;
 
     .fc-item__container.u-faux-block-link--hover {
-        background-color: $special-report-alt-faded !important;
+        background-color: $special-report-alt-faded;
```

Comment:
`!important` is always a bad idea. In this particular case it was overriding the desired colour of the card (hover and non-hover state) when a `SpecialReportAlt` card was added to a container with a palette. Palette colours should take precedence but this wasn't happening.

<img width="416" alt="image" src="https://user-images.githubusercontent.com/19683595/225913294-6e8818bb-b6f7-4ce9-a040-46d3448c01a0.png">